packages/rustworkx/meta.yaml

```diff
@@ -0,0 +1,26 @@
+package:
+  name: rustworkx
+  version: 0.17.0a3
+  top-level:
+    - rustworkx
+  tag:
+    - rust
+source:
+  url: https://github.com/IvanIsCoding/rustworkx/releases/download/v0.17.0a3/rustworkx-0.17.0a3.tar.gz
+  sha256: 8bd0c295134e2b0c03808d4e69428b41153849db6488839084a78793c337f191
+build:
+  script: export RUSTFLAGS="-C target-feature=+atomics,+bulk-memory,+mutable-globals -C link-arg=-s -C link-arg=ALLOW_MEMORY_GROWTH=1 -C link-arg=-s -C link-arg=MODULARIZE=1 -C link-arg=-s -C link-arg=EXPORT_NAME=createModule -C link-arg=-s -C link-arg=EXPORT_ES6=1 -C link-arg=-s -C link-arg=ASSERTIONS=1"
```

Comment:
I think you should get rid of this `RUSTFLAGS` setting. It surely won't with this.
```suggestion
```
If you do need to add something to `RUSTFLAGS` try to do it like this:
```
   export RUSTFLAGS="$RUSTFLAGS -C <some new flag>"
```
The stuff that is in there by default is important.

Reply:
The reality is the build just need s`"-C", "target-feature=+atomics,+bulk-memory,+mutable-globals"` in addition to the default flags. But for `pyodide build` setting `RUSTFLAGS` doesn't seem to be additive?

Reply:
You can do `export RUSTFLAGS="$RUSTFLAGS -Ctarget-feature=+atomics,+bulk-memory,+mutable-globals`. But `bulk-memory` and `mutable-globals` are on by default. Atomics probably isn't.